astropy/stats/funcs.py

```diff
@@ -1284,7 +1284,10 @@ def _kraft_burrows_nousek(N, B, CL):
     '''
     from astropy.utils.compat.optional_deps import HAS_SCIPY, HAS_MPMATH
 
-    if HAS_SCIPY and N <= 100:
+    if HAS_SCIPY:
+        if N > 100:
```

Comment:
This doesn't work. If `scipy` is installed then the `mpmath` check is never run.

If you want to eliminate OverflowError from the trace, this PR could instead add a ``from None`` to the original error (line 1292 of the unmodified file).

Reply:
Hi, yes sorry about that. That indeed was an oversight. Will update the PR and let you know...